src/Shared/Recruit.Vacancies.Client/Infrastructure/EventHandlers/VacancyClosedEventHandler.cs

```diff
@@ -46,6 +46,12 @@ public class VacancyClosedEventHandler : INotificationHandler<VacancyClosedEvent
 
         public async Task Handle(VacancyClosedEvent notification, CancellationToken cancellationToken)
         {
+            var vacancy = await _repository.GetVacancyAsync(notification.VacancyId);
+            if(vacancy.ClosureReason == ClosureReason.BlockedByQa)
```

Comment:
We need to include closure reasons `TransferredByQa` and `TransferredByEmployer` in here as well.